packages/@aws-cdk-containers/ecs-service-extensions/test/test.datadog.ts

```diff
@@ -0,0 +1,155 @@
+import { expect, haveResource } from '@aws-cdk/assert';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as secretsManager from '@aws-cdk/aws-secretsmanager';
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import { DatadogAgent, Container, Environment, Service, ServiceDescription } from '../lib';
+
+export = {
+  'should be able to add Datadog agent extension to a service'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+
+    // WHEN
+    const SECRET_ARN = 'arn:aws:secretsmanager:us-west-2:209640446841:secret:datadog-api-key-secret-JxST1g';
+
+    // Import a secret from the account.
+    const datadogApiKey = secretsManager.Secret.fromSecretAttributes(stack, 'datadog-api-key-secret', {
+      secretCompleteArn: SECRET_ARN,
+    });
+
+    const environment = new Environment(stack, 'production');
+    const serviceDescription = new ServiceDescription();
+
+    serviceDescription.add(new Container({
+      cpu: 256,
+      memoryMiB: 512,
+      trafficPort: 80,
+      image: ecs.ContainerImage.fromRegistry('nathanpeck/name'),
```

Comment:
Can you make this ecr registry generic (not referencing your personal username)? This applies throughout the integ test code